force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +363,14 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
-
+        Integer k = 0;
         // Loop through the list of integers
-            // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
+        for (Integer i : numbers) {
+            if (i >= 1) {
+                 k += i;
```

Comment:
Try not to use single letters as variable. Make your variable names meaningful so if someone looks at it they can tell what it does. 
Use the continue keyword to skip an iteration
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_loops.htm